### PR TITLE
heimdal 7.7.0

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Previous attempts: #50179, #41032

Args added based on https://github.com/heimdal/heimdal/blob/master/.travis.yml

### CODE REVIEW

File: Formula/heimdal.rb [hunk 1]
```diff
@@ -14,6 +13,11 @@ class Heimdal < Formula
 
   keg_only :shadowed_by_macos, "macOS provides Kerberos"
 
+  depends_on "berkeley-db"
+  depends_on "bison"
+  depends_on "flex"
+  depends_on "lmdb"
+  depends_on "openldap"
```
Comment (claui) [inline Formula/heimdal.rb:20]: Are those all runtime dependencies?

Comment (SeekingMeaning) [reply Formula/heimdal.rb:20]: I'm honestly not sure haha

Comment (SeekingMeaning) [reply Formula/heimdal.rb:20]: Okay so I tested `brew linkage --test heimdal` and it looks like bison can be marked as a compile-time dependency

**Review by claui — APPROVED** [review]

**Comment by claui** [comment]
Thanks @SeekingMeaning! 🍻

**Pull request closed without merging**